Formula/p/procenv.rb

```diff
@@ -0,0 +1,16 @@
+class Procenv < Formula
+  desc "Show process environment variables"
+  homepage "https://github.com/henrik242/procenv"
+  url "https://github.com/henrik242/procenv/archive/refs/tags/1.1.tar.gz"
+  sha256 "7371164f700c695fe27783b4471c2817e229fa7acc20d7a8b37a49687ef5f65d"
+  license "MIT"
+
+  def install
+    system "make"
+    bin.install "procenv"
+  end
+
+  test do
+    system "#{bin}/procenv"
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.